flake.nix

```diff
@@ -57,8 +55,6 @@
           in
             builtins.filterSource filter ./.;
         templates.mkflake.description = "template with necessary folders for mkFlake usage";
-      } // {
-        inherit lib;
```

Comment:
How did the `lib` diffs get in here? lib gets exported in `mkFlake`, and its necessary to do it that way, so lib functions have access to to the right `self`. I'll apply the other change manually though, `//` is the right thing to do.

Reply:
although I don't know about the syntax here, I think what this will end up doing is merge the arguments first and pass everything to `mkFlake`, since the `//` operator has priority over function arguments.

Reply:
nevermind it does seem to work